avatars/avatars.js

```diff
@@ -1971,12 +1985,18 @@ class Avatar {
       this.debugMesh.visible = debug.enabled;
     }
   }
-
   isAudioEnabled() {
-    return !!this.microphoneWorker;
+    return !!this.audioWorker;
+  }
+  isMicrophoneEnabled() {
+    return this.localPlayer && !!this.microphoneWorker;
   }
   setAudioEnabled(enabled) {
```

Comment:
IMO this should be split into the two distinct pipeline cases, and two methods. Otherwise it's confusing that multiple workers (some of which are not used) are created.

The caller should call either `setLocalAudioEnabled` or `setRemoteAduioEnabled`.

The two methods could still share the `audioRecognizer` which is created/deleted on demand.